src/main.cpp

```diff
@@ -1,9 +1,133 @@
-#include <cstdlib>
+#include <algorithm>
+#include <iostream>
+#include <iterator>
 #include <iostream>
+#include <cstdlib>
+#include <fstream>
+#include <sstream>
+#include <iomanip>
+#include <string>
+#include <vector>
+using namespace std;
+
+// Global Variables & Functions
+vector<string> all_types = {"PROGRAM", "VAR", "END_VAR", "END_PROGRAM", "BOOL", "TRUE", "FALSE"}; //keywords
+vector<string> operators = {"<", ">", "+", "-", "%", ":="}; // operators
+vector<string> all_sc = {"{", "}", "[", "]", ";", ":"}; // special characters
+void tokenizerFunc (string sample_input); // main lexer function
+typedef struct Token{ // where/how the keywords are going to be saved
+    string key;
+    string value;
+}TOTO;
```

Comment:
What is TOTO?

Reply:
Temporary insignificant name of the vector. I've renamed it.